docs/platform/howto/attach-vpc-aws-tgw.md

```diff
@@ -1,130 +1,170 @@
+      1. On the **Virtual private clouds** page, select your organization VPC.
+      1. On the **VPC details** page, go to the **Overview** section, and copy **ID**.
+   </TabItem>
+   </Tabs>
 
-Use [`avn vpc list`][avnvpc] to find the ID for your Project VPC. The
-`project_vpc_id value` (a UUID4 string) will be referred to as
-`$project_vpc_id` later.
+   The Aiven VPC ID will be referred to as `$vpc_id`.
 
-### Determine the IP ranges to route from the Project VPC to the AWS Transit Gateway
+1. Determine the IP ranges to route from the VPC to the AWS Transit Gateway.
 
-While a Transit Gateway has a route table of its own, and will by
-default route traffic to each attached network (directly to attached VPC
-or indirectly via VPN attachments), the attached route tables of the VPC
-need to be updated to include the TGW as a target for any IP range
-(CIDR) that should be routed using the VPC attachment.
+   :::note
+   A Transit Gateway has a route table of its own and, by default, routes traffic to each
+   attached network (directly to an attached VPC or indirectly through VPN attachments).
+   Attached route tables of the VPC need to be updated to include the TGW as a target for
+   any IP range (CIDR) to be routed using the VPC attachment.
+   :::
 
-These IP ranges
-must be configured when creating the attachment for an Aiven Project
-VPC. The IPv4 range will be referred below to as
-`$user_peer_network_cidr`.
+   The IP ranges will be referred to as `$user_peer_network_cidr`.
 
-### Create Aiven peering connection
+## Request the attachment
 
-A Transit Gateway VPC attachment is created by making a request to the
-Aiven API for a peering connection. The Aiven API handles both actual
-[AWS VPC peering
-connections](https://docs.aws.amazon.com/vpc/latest/peering/what-is-vpc-peering)
-and AWS Transit Gateway VPC attachments as peering connections.
+### Send the connection request
 
-```shell
+To create the Transit Gateway - VPC attachment, make a request to the Aiven API for a
+peering connection.
+
+:::note
+In the request, you can use the `--user-peer-network-cidr` argument multiple times to
+define more than one peer network CIDR. You can also create the attachment without a CIDR,
+but you would need to add it later to make your attachment operational.
+:::
+
+Depending on the type of your Aiven VPC, select **Project VPC** or **Organization VPC**,
+and run the provided commands, replacing `$` placeholders with meaningful values.
+
+   <Tabs groupId="group1">
+   <TabItem value="pj-vpc" label="Project VPC" default>
+```bash
 avn vpc peering-connection create       \
-  --project-vpc-id $aws_vpc             \
+  --project-vpc-id $vpc_id              \
   --peer-cloud-account $user_account_id \
   --peer-vpc $user_tgw_id               \
   --user-peer-network-cidr $user_peer_network_cidr
 ```
+   </TabItem>
+   <TabItem value="org-vpc" label="Organization VPC">
+```bash
+avn organization vpc peering-connection create \
```

Comment:
This one (and organization VPC commands in general) will require the organization to be given with `--organization-id` as it's a part of the path of the resource. For project VPC commands `--project` is optional as there's in the CLI the concept of a default project set with `avn project switch`, but there's no concept of a default organization, and thus `--organization-id` needs to be explicitly given.